components/brave_shields/browser/filter_list_catalog_entry.h

```diff
@@ -41,9 +45,16 @@ class FilterListCatalogEntry {
   std::string title;
   std::vector<std::string> langs;
   std::string support_url;
+  std::string desc;
+
+  // Optional fields with default values
```

Comment:
can you add some comments explaining these?

Reply:
They are primarily documented at https://github.com/brave/adblock-resources#filter-list-description-format. I'll add that link in a comment.